src/main/java/com/zufar/onlinestore/payment/enums/PaymentConstants.java

```diff
@@ -31,6 +31,8 @@ public enum PaymentStatus {
     PAYMENT_IS_SUCCEEDED("payment_intent.succeeded",
             "Payment is succeeded");
 
+    public static final String CURRENCY = "usd";
```

Comment:
It is paymentStatus. paymentStatus is not good place for currency

Reply:
Done